src/components/sheetSettings.jsx

```diff
@@ -126,27 +248,65 @@ class Settings extends Component {
                     </select>
                 </div>
                 {this.displayCustomInputs()}
-    
+
                 <button
+                    className="applyButton"
                     onClick={() => {
                         this.saveSettings();
                     }}
                     disabled={!this.state.formChange}
                 >
                     Apply
                 </button>
-                    <hr />
+                <hr />
                 <button
+                    className="exportButton"
                     onClick={() => {
                         window.print();
                     }}
                 >
                     Export as pdf
                 </button>
+                <button
+                    className="exportButton"
+                    onClick={() => {
+                        this.saveHtml();
+                    }}
+                >
+                    Export as HTML
+                </button>
+                <button
+                    className="exportButton"
+                    onClick={() => {
+                        this.saveSettings();
+                        this.saveJson();
+                    }}
+                >
+                    Export as JSON
+                </button>
+
+                <hr />
+
+                <button
+                    onClick={() => {
+                        document.getElementById("importJson").click();
```

Comment:
We can eliminate the `document.getElementById` calls by using React refs instead, in this case it would look like this:

![image](https://github.com/naturalcrit/Axe/assets/19826659/02c5bee6-02f7-4ee4-a7e8-44ddf7d85b01)
![image](https://github.com/naturalcrit/Axe/assets/19826659/862e28c5-7203-4be6-9596-5a74c90ad286)
![image](https://github.com/naturalcrit/Axe/assets/19826659/3387b62a-2c0b-4570-bff6-190d46b5afda)

Reply:
then grab the file like this?

<img width="467" alt="image" src="https://github.com/naturalcrit/Axe/assets/91662974/89304aee-3cfc-47e1-ada8-add87cab1986">

Reply:
Exactly